packages/web/components/bridge/immersive/amount-and-review-screen.tsx

```diff
@@ -159,7 +159,7 @@ export const AmountAndReviewScreen = observer(
                     quote={quote}
                     onCancel={goBack}
                     onConfirm={() => {
-                      quote.onTransfer();
+                      quote.onTransfer().catch(noop);
```

Comment:
Personally, I added the catch in the `useBridgeQuotes` hook instead. And logged the error